src/features/moviesSlice.js

```diff
@@ -21,14 +21,14 @@ const moviesSlice = createSlice({
         setLastPage: (state, { payload: totalPages }) => {
             state.page = totalPages;
         },
-        fetchPopularMovies: state => {
+        fetchPopular: state => {
             state.status = "loading";
         },
         setMoviesData: (state, { payload: data }) => {
             state.moviesData = data;
-            state.status = undefined;
+            state.status = "success";
         },
-        fetchPopularMoviesError: state => {
```

Comment:
In 31 code line name was changed with fetchPopularMoviesError to fetchPopulaError. In 47 code line is declare again fetchPopularMoviesError.

Reply:
Thanks, I fixed it now :D